journeyStar_shacl.ttl

```diff
@@ -2,11 +2,8 @@
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix currencyA: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/> .
-@prefix currency: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/>.
 @prefix schema: <https://schema.org/> .
 @prefix dbr: <http://dbpedia.org/resource/> .
-@prefix dbo: <http://dbpedia.org/ontology/> .
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix js-shacl: <http://journey-star.dhlab.unibas.ch/shacl/trip-shacl#> .
```

Comment:
I would suggest to change the namespace to `journey-shacl` for consistency's sake